cmd/codeqlExecuteScan.go

```diff
@@ -172,6 +172,12 @@ func uploadResults(config *codeqlExecuteScanOptions, utils codeqlExecuteScanUtil
 }
 
 func runCodeqlExecuteScan(config *codeqlExecuteScanOptions, telemetryData *telemetry.CustomData, utils codeqlExecuteScanUtils) error {
+	codeqlVersion, err := os.ReadFile("/etc/image-version")
+	if err != nil {
+		log.Entry().Infof("CodeQL image version: unknown")
+	}
+	log.Entry().Infof("CodeQL image version: %s", string(codeqlVersion))
+
```

Comment:
Should there be an "else"? We do not want both in the case the file is absent, right?

Reply:
Sure, I missed that bit, thank you. Fixed it.